qml/pages/GalleryUI.qml

```diff
@@ -94,24 +105,42 @@ Page {
             height: parent.height
             color: 'black'
 
-            Image {
+            Thumbnail {
                 id: thumbnail
 
-                asynchronous: true
+                width: parent.width
```

Comment:
I think we don't need to set width and height when we already have "anchors.fill: parent" below.

Reply:
True, will do